contracts/RiskManagerV1.sol

```diff
@@ -172,6 +223,21 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         auctionLengthChangeInitiated = 0;
     }
 
+    /// @notice Get the time remaining until the collateralization threshold
+    ///         can be updated.
+    /// @return Remaining time in seconds.
+    function getRemainingCollateralizationThresholdUpdateTime()
+        external
+        view
+        returns (uint256)
+    {
+        return
+            getRemainingChangeTime(
+                collateralizationThresholdInitiated,
+                GOVERNANCE_TIME_DELAY
```

Comment:
We have multiple governance delay constants. Perhaps we could name it:
`COLLATERALIZATION_THRESHOLD_GOVERNANCE_DELAY` ?

Also, if I remember correctly, it was suggested to keep governance-related consts in `CoveragePoolConstants`.
Am I right, @pdyraga?

Reply:
I think `GOVERNANCE_TIME_DELAY` is fine in this case because all risk manager governable parameters will have [the same delay of 12 hours](https://github.com/keep-network/coverage-pools/blob/main/docs/design.adoc#governance). If they would have different values, then sure, using different names would make sense.

> Also, if I remember correctly, it was suggested to keep governance-related consts in `CoveragePoolConstants`.

Only if they belong to coverage pools world, Risk Manager is a kind of a plug-in in this system, so I think what's been proposed here and earlier in the auction length update PR makes sense.